### ISSUE TITLE
Text overlapping Access tokens tab and Got feedback tab at 200% zoom

### ISSUE THREAD

**Issue by Onokaev** [microsoftgraph/microsoft-graph-explorer-v4#1392]
**Describe the bug**
At 200% zoom, the text on the Access tab and Got feedback tab overlap.
![image](https://user-images.githubusercontent.com/45680252/151932038-e102d7a8-ed9d-4b4d-9eeb-5e46bf08fb3c.png)

**To Reproduce**
1. Go to GE
2. Magnify screen to 200%
3. Check on the Access token tab and Got feedback tab

**Expected behavior**
The text on each tab should be readable

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
